pkg/virt-handler/vm_test.go

```diff
@@ -1684,15 +1686,14 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Expect(len(arg.(*v1.VirtualMachineInstance).Status.Interfaces)).To(Equal(1))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].Name).To(Equal(podCacheInterface.Iface.Name))
 				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IP).To(Equal(podCacheInterface.PodIP))
-				Expect(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs[0]).To(Equal(podCacheInterface.PodIP))
-
+				Expect(reflect.DeepEqual(arg.(*v1.VirtualMachineInstance).Status.Interfaces[0].IPs, podCacheInterface.PodIPs)).To(BeTrue())
```

Comment:
[Equal](https://onsi.github.io/gomega/#equalexpected-interface) already uses `reflect.DeepEqual`, please use it instead.

Reply:
wont it be too implicit ? and we strive to explicit

Reply:
done (the ditto as well)